doc/COOKBOOK-FIXERS.md

```diff
@@ -128,7 +128,7 @@ final class RemoveCommentsFixerTest extends AbstractFixerTestCase
 
     public function provideFixCases()
     {
-        return array();
+        return [];
```

Comment:
This is getting quite exhausting to me for just few docs lines to fix. Back and forth.

Could you finish this in merging please?